build/ci/templates/dotnet-mobile-install-windows.yml

```diff
@@ -0,0 +1,30 @@
+parameters:
+  - name: UseNet6SDK
+    default: 'false'
+
+steps:
+
+  - ${{ if eq(parameters['UseNet6SDK'], 'true') }}:
+    - script: |
+        cp global.json global-net7.json
+        cp global-net6.json global.json
+      displayName: Replace global.json with .NET 6
+
```

Comment:
@jeromelaban This step isn't run when I pass `UseNet6SDK` as true.

I'm not sure what the right syntax is and/or how can I investigate conditions that are not met.

Can you help with that?

Reply:
Resolved